tee-worker/cli/src/trusted_base_cli/commands/query_storage.rs

```diff
@@ -0,0 +1,73 @@
+use crate::{command_utils::get_worker_api_direct, trusted_cli::TrustedCli, Cli};
+use codec::Decode;
+use itc_rpc_client::direct_client::DirectApi;
+use itp_rpc::{RpcRequest, RpcResponse, RpcReturnValue};
+use itp_types::DirectRequestStatus;
+use itp_utils::FromHexPrefixed;
+use log::{debug, error, warn};
+use my_node_runtime::Hash;
+
+use std::sync::mpsc::channel;
+
+#[derive(Parser)]
+pub struct QueryStorageCommand {
+	/// Storage Key
+	module: String,
+
+	storage: String,
+}
+impl QueryStorageCommand {
+	pub(crate) fn run(&self, cli: &Cli, trusted_args: &TrustedCli) {
```

Comment:
maybe rename the function name to `query`?

Reply:
Looks like `run` is a name for consistency.